script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -165,32 +165,32 @@ In fact, I'm grateful to you for treating me as your brother, even after I lived
 I have made it clear that to interfere in this matter is to go against me, and by extension, against the Tohno family."
 <Page48>
 "――――――"
-I can't help but look at her in amazement.
+Unconsciously, my eyes get wide.//　思わず目を見張る。
 Even Makihisa couldn't reject the branch families. But for Akiha to persuade all of them... no, maybe she argued them down.
-...Yeah.
-She has grown up well in appearance, but I wonder how much she has grown up inside as well. As the older brother, I'm a little scared of my sister's rapid development.
+...Well.//　……ううん。
+She has grown up well in appearance, but I wonder how much she has grown up inside as well. The family registrar says I'm her older brother and I'm a little scared of my sister's rapid development.//　外見も立派に育ったけど、その中身もどれだけの成長を果たしたのか。戸籍上・兄として、妹の急成長がちょっと怖い。JP has some weird formatting at the end.
```

Comment:
I guess it's a callback to earlier where she mentions it? It it supposed to be a bit of snark?

Reply:
That's how I read it anyways